bson/src/main/org/bson/codecs/UuidCodec.java

```diff
@@ -20,22 +20,23 @@
 import org.bson.BsonBinary;
 import org.bson.BsonBinarySubType;
 import org.bson.BsonReader;
+import org.bson.BsonSerializationException;
 import org.bson.BsonWriter;
 import org.bson.UuidRepresentation;
-
 import org.bson.internal.UuidHelper;
 
 import java.util.UUID;
 
+import static org.bson.assertions.Assertions.notNull;
+
 /**
  * Encodes and decodes {@code UUID} objects.
  *
  * @since 3.0
  */
 public class UuidCodec implements Codec<UUID> {
 
-    private final UuidRepresentation encoderUuidRepresentation;
-    private final UuidRepresentation decoderUuidRepresentation;
+    private final UuidRepresentation uuidRepresentation;
```

Comment:
Just a simplification to implementation.  No behavioral change.

Reply:
Ack